.github/SECURITY.md

```diff
@@ -4,27 +4,27 @@
 
 Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/Microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [our GitHub organizations](https://opensource.microsoft.com/).
 
-If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
+If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](<https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)>), please report it to us as described below.
```

Comment:
Me neither, especially that it's done only on one link.

Reply:
I'm guessing it's because the link contains parentheses, but I don't know where to find evidence of this.

Reply:
Only documentation I found on this is https://www.markdownguide.org/basic-syntax/#link-best-practices. Can you try %28 and %29 instead of ( and )? Then the angle brackets shouldn't be required.

Reply:
Other suggestions and an overview of the problem can be found here: https://meta.stackexchange.com/questions/13501/links-to-urls-containing-parentheses

Reply:
```suggestion
If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383%28v=technet.10%29), please report it to us as described below.
```

Reply:
This solution have worked.